package.json

```diff
@@ -24,6 +24,16 @@
     "email": "joel@oblador.se"
   },
   "homepage": "https://github.com/oblador/react-native-store-review",
+  "devDependencies": {
+    "@types/react": "^16.9.19",
+    "@types/react-native": "0.62.13",
+    "react": "16.11.0",
+    "react-native": "0.62.2"
+  },
+  "peerDependencies": {
+    "react": "*",
```

Comment:
It's correct to list these as peerDependencies, but (at least historically) it's been causing a bunch of warnings in peoples consoles despite them having reasonable versions of it. I would doubt that anyone would install this package and not have react/react-native installed.

Reply:
You are right!